Assets/Scripts/Scenes/InGame/Player/Item/ItemEffectScript/RlyehTextEffect.cs

```diff
@@ -19,19 +21,40 @@ public override void OnThrow()
 
     public override void Effect()
     {
-        if(_used) return;
-        _used = false;
-        var fog = Instantiate(_fogEffect, ownerPlayerStatus.transform.position,Quaternion.identity);
-        GameObject[] Enemys = GameObject.FindGameObjectsWithTag("Enemy");
+        if (_used) return;
+        _used = true;
+        _audioSource = GetComponent<AudioSource>();
+        if (_audioSource == null)
+        {
+            _audioSource = gameObject.AddComponent<AudioSource>();
+        }
+        _audioSource.PlayOneShot(_rlyehTextSound);
```

Comment:
```suggestion
                SoundManager.Instance.PlaySe("se_R'lyehText_00",transform.position);
```
SEの鳴らし方は統一しているため、こちらの関数を使用してください